closure/goog/net/xhriopool.js

```diff
@@ -65,6 +78,9 @@ goog.net.XhrIoPool.prototype.createObject = function() {
       xhrIo.headers.set(key, value);
     });
   }
+  if (this.withCredentials_) {
+    xhrIo.setWithCredentials(true);
```

Comment:
Thanks for the comments. I'll look into this. Not sure I have time to do it this week though.